### PR TITLE
[vcpkg baseline][openvino] Don't pick OpenCV; cleanup

### PR DESCRIPTION
**Pull request by dg0yt**
Fixes installation order bug.

### CODE REVIEW

**Comment by dg0yt** [comment]
Port bug, not just cleanup. And not an insignificant bug: blocking other PRs now.

**Comment by jimwang118** [comment]
All feature test passed with x64-windows triplet.

**Review by sandye51 — APPROVED** [review]

File: ports/openvino/portfile.cmake [hunk 1]
```diff
@@ -118,21 +119,25 @@ if(CMAKE_HOST_WIN32)
...
+        "-DENABLE_PYTHON=OFF"
         "-DENABLE_SAMPLES=OFF"
+        "-DENABLE_SYSTEM_PUGIXML=ON"
+        "-DENABLE_SYSTEM_TBB=ON"
+        "-DENABLE_TBBBIND_2_5=OFF"
         "-DENABLE_TEMPLATE=OFF"
-        "-DENABLE_PYTHON=OFF"
-        "-DCPACK_GENERATOR=VCPKG"
-        "-DENABLE_JS=OFF"
+        "-DPKG_CONFIG_EXECUTABLE=${PKGCONFIG}"
```
Comment (sandye51) [inline ports/openvino/portfile.cmake:140]: I think we can also disable https://cmake.org/cmake/help/latest/module/FindWget.html (used here https://github.com/openvinotoolkit/openvino/blob/releases/2024/5/cmake/developer_package/download/download_and_check.cmake#L5)

File: ports/openvino/portfile.cmake [hunk 2]
```diff
@@ -118,21 +119,25 @@ if(CMAKE_HOST_WIN32)
     list(APPEND FEATURE_OPTIONS "-DENABLE_API_VALIDATOR=OFF")
 endif()
 
+vcpkg_find_acquire_program(PKGCONFIG)
```
Comment (sandye51) [inline ports/openvino/portfile.cmake:122]: is it possible to disable some pkg-config modules search like it's done for cmake ones?

File: ports/openvino/portfile.cmake (hunk 1 above)
Comment (dg0yt) [reply ports/openvino/portfile.cmake:140]: This PR is a minimal fix for current regression. 

(wget in CMake, seriously? :facepalm:)

File: ports/openvino/portfile.cmake (hunk 2 above)
Comment (dg0yt) [reply ports/openvino/portfile.cmake:122]: No.

**Pull request merged**